config/bin_version_strings.cfg

```diff
@@ -295,10 +295,16 @@ iptunnel;;unknown;"iptunnel\ [0-9](\.[0-9]+)+?";"sed -r 's/iptunnel\ ([0-9](\.[0
 ipunz;;commercial;"^ipunz V[0-9]\.[0-9]$";"sed -r 's/ipunz\ V([0-9](\.[0-9]+)+?)$/ipcomm:ipunz:\1/'";
 ipzip;;commercial;"^ipzip V[0-9]\.[0-9]$";"sed -r 's/ipzip\ V([0-9](\.[0-9]+)+?)$/ipcomm:ipzip:\1/'";
 iputils;;unknown;"iputils-s{,3}[0-9]+";"sed -r 's/iputils-s{,3}([0-9]+)$/iputils:\1/'";
-isc-dhclient;;ISC;"Internet\ Systems\ Consortium\ DHCP\ Client\ [0-9](\.[0-9]+)+?([a-z][0-9])?$";"sed -r 's/Internet\ Systems\ Consortium\ DHCP\ Client\ ([0-9](\.[0-9]+)+?([a-z][0-9])?)$/isc:dhcp_client:\1/'";
-isc-dhclient;;ISC;"isc-dhclient-[0-9](\.[0-9]+)+?$";"sed -r 's/isc-dhclient-([0-9](\.[0-9]+)+?)$/isc:dhcp_client:\1/'";
-isc-dhclient;;ISC;"isc-dhclient-[0-9](\.[0-9]+)+?-([ABPabp]|rc|RC)[0-3]$";"sed -r 's/isc-dhclient-([0-9](\.[0-9]+)+?(-([ABPabp]|rc|RC)[0-3])?)$/isc:dhcp_client:\1/'";
-isc-dhclient;;ISC;"isc-dhclient-[0-9](\.[0-9]+)+?-ESV-R[0-9]$";"sed -r 's/isc-dhclient-([0-9](\.[0-9]+)+?(-ESV-R[0-9])?)$/isc:dhcp_client:\1/'";
+isc-dhclient;;ISC;"Internet\ Systems\ Consortium\ DHCP\ Client\ [0-9](\.[0-9]+)+?([a-z][0-9])?$";"sed -r 's/Internet\ Systems\ Consortium\ DHCP\ Client\ ([0-9](\.[0-9]+)+?([a-z][0-9])?)$/dhcp_client:\1/'";
+isc-dhclient;;ISC;"isc-dhclient-[0-9](\.[0-9]+)+?$";"sed -r 's/isc-dhclient-([0-9](\.[0-9]+)+?)$/dhcp_client:\1/'";
+isc-dhclient;;ISC;"isc-dhclient-[0-9](\.[0-9]+)+?-([ABPabp]|rc|RC)[0-3]$";"sed -r 's/isc-dhclient-([0-9](\.[0-9]+)+?(-([ABPabp]|rc|RC)[0-3])?)$/dhcp_client:\1/'";
+isc-dhclient;;ISC;"isc-dhclient-[0-9](\.[0-9]+)+?-ESV-R[0-9]$";"sed -r 's/isc-dhclient-([0-9](\.[0-9]+)+?(-ESV-R[0-9])?)$/dhcp_client:\1/'";
+isc-dhcpd;;ISC;"Internet\ Systems\ Consortium\ DHCP\ Server\ [0-9](\.[0-9]+)+?([a-z][0-9])?$";"sed -r 's/Internet\ Systems\ Consortium\ DHCP\ Server\ ([0-9](\.[0-9]+)+?([a-z][0-9])?)$/dhcp:\1/'";
+isc-dhcpd;;ISC;"Internet\ Systems\ Consortium\ DHCP\ Server\ [0-9](\.[0-9]+)+?([a-z][0-9])?$";"sed -r 's/Internet\ Systems\ Consortium\ DHCP\ Server\ ([0-9](\.[0-9]+)+?([a-z][0-9])?)$/dhcpd:\1/'";
+isc-dhcpd;;ISC;"isc-dhcpd-[0-9](\.[0-9]+)+?$";"sed -r 's/isc-dhcpd-([0-9](\.[0-9]+)+?)$/dhcp:\1/'";
```

Comment:
wondering on these rules as they do not fit the results on my strings output from your binary. They probably match the emulation output (s115) but not teh static checks (s09). Why not using the multi-grep rule from [here](https://github.com/e-m-b-a/emba/issues/1155#issuecomment-2114150647) for static checking?

Reply:
btw. ist this a public firmware you are using for testing? Can you provide a download link?

Reply:
> wondering on these rules as they do not fit the results on my strings output from your binary. They probably match the emulation output (s115) but not teh static checks (s09). Why not using the multi-grep rule from [here](https://github.com/e-m-b-a/emba/issues/1155#issuecomment-2114150647) for static checking?

Indeed, as far as the binaries attached to the issue are concerned, strings can only be found through emulation as static analysis won't find them as is in the binary.

There might be other build of the same binaries in the wild that do include those whole strings for static check. I don't know. But given that we don't have such samples for now, I could add `no_static` to the rules. 

I'll also check for `multi_grep` and re-analyze before committing. 

That firmware is unfortunately not public. I also tried to create a mini-firmware version by keeping only a a few generic binaries that could be shared (also because as it stand now, even when excluding S13, S14 and S16 from sbom-default it still takes 9 hours on a 32-core system), but emulation would never run. Can't tell why.

Reply:
ok, for static analysis it is fine and the dynamic strings are not much different to the already known. So, I think we are fine after adjusting.
"multi_grep" is currently only available in static mode. It is on the todo list for dynamic analysis as well as combining multiple rules (no_static, multi_grep and further rules in the future)